### ISSUE TITLE
Updated to containerd.io/focal 1.4.8-1 in Ubuntu 20.04.2 LTS broke Rancher 

### ISSUE THREAD

**Issue by apulache** [rancher/rancher#33718]
**Rancher Server Setup**
- Rancher version: 2.5.8
- Installation option (Docker install/Helm Chart):
- Docker
  Client: Docker Engine - Community
 Version:           20.10.7
 API version:       1.41
 Go version:        go1.13.15
 Git commit:        f0df350
 Built:             Wed Jun  2 11:56:38 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.7
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       b0f5bc3
  Built:            Wed Jun  2 11:54:50 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.6
  GitCommit:        d71fcd7d8303cbf684402823e425e9dd2e99285d
 runc:
  Version:          1.0.0-rc95
  GitCommit:        b9ee9c6314599f1b4a7f497e1f1f856fe433d3b7
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

- OS: Ubuntu 20.04.2 LTS

**Describe the bug**
After updated to containerd.io/focal 1.4.8-1 using apt update/upgrade Rancher is constantly re-starting

**To Reproduce**
Start with Ubuntu 20.04.2 LTS with Rancher 2.5.8 installed on
sudo apt update
sudo apt upgrade -y
Make sure containerd.io/focal 1.4.8-1 is installed

**Result**
Rancher is constantly re-starting

**Expected Result**
Rancher should start and function properly

**Additional context**
containerd.io/focal 1.4.8-1 was pull from Ubuntu repositories

**Comment by superseb** [comment]
What is in the log? Did you restart Docker? Does any container run (`docker run -d nginx`)?

**Comment by apulache** [comment]
Hi Superseb.

I cannot run additional tests any longer as after Rancher failed to restart I've restore a previous functioning snapshot on my system.

Before I did it I've stopped and remove Rancher container and then Rancher image and I've reinstalled it with the latest image. Docker with the the updated Containerd pull the image and try to start the new Rancher container but it never properly ran and falled in a restarting continuous cycle.

Thanks in advance.

**Comment by mrulke** [comment]
i am having the same issue

**Comment by mrulke** [comment]
dont know where to get the logs from?

**Comment by superseb** [comment]
Where is the `rancher/rancher` container in `Restarting` state in `docker ps`? According to the processes it is running, is this single or HA? According to what from the initial post is this the same issue?

**Comment by mrulke** [comment]
I set this up as a docker container. So I'm really lost on where it went and how it is still trying to start. when docker ps brings back nothing

**Comment by superseb** [comment]
`docke ps -a` shows all containers, it probably is shown in there. After you located it, you can use `docker logs CONTAINER_NAME` to check it's logs.

**Comment by mrulke** [comment]
docker ps -a brings back
`CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
root@devserver:/home/simon# `

I'm lost on what is happening. don't want to have to rebuild as my kube cluster is still running I just cant access any of the nodes.

**Comment by jsmythsci** [comment]
We seem to be encountering the same issue on Ubuntu 18.04.5 LTS.

Rancher was working fine until we applied all pending patches this morning. Included in the patch list is containerd.io v1.4.8-1. After patching, the Rancher container just keeps restarting. Exporting the volumes and starting a new container did not help. Running a brand new Rancher container (after stopping the original) also fails with the same restart behaviour.

Downgrading containerd.io to v1.4.6-1 (the version that we upgraded from) also did not help. Either there are other packages that also need to be downgraded or the upgrade process broke something that the downgrade process did not repair.

We finally recovered by restoring the host VM from snapshot but this is obviously not a long-term solution.

I will do some more testing on this to see if I can pinpoint the exact (set of) package(s) that cause this issue to start for us.

If anyone has any insight on where we might want to start the next steps of our investigation, that would be appreciated.
